addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -395,6 +395,24 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Focus inside the editable area of the editor.
+     * Set cursor at the end of the text, ready to type.
```

Comment:
Suggestion not mandatory:
Could you update the comment to include the other case you added (set the history to last step).

Also, could you separate the "short description" to the "longer description" ?

It it's ok with you:
```
    /**
     * Focus inside the editor.
     *
     * Set cursor to the editor latest position before blur or to the last editable node, ready to type.
     */
```